setup/common/003_solaris_cert_fix.rb

```diff
@@ -0,0 +1,54 @@
+test_name 'Add digicert to solaris keystore'
+
+# Only need to run this on soliars 11, 11.2, and sparc
+skip_test 'No solaris 11 version that needed keystore updating' if ! hosts.any? { |host| host.platform=~ /solaris-11(\.2)?-(i386|sparc)/}
+
+solaris_hosts = Array.new
+
+hosts.each do |host|
+  solaris_hosts.push(host) if host.platform=~ /solaris-11(\.2)?-(i386|sparc)/
+end
+
+step 'Download the DigiCert Trusted Root G4' do
+  solaris_hosts.each do |host|
+    on(host, 'curl --remote-name https://cacerts.digicert.com/DigiCertTrustedRootG4.crt.pem')
+  end
+end
+
+step 'Validate cert' do
```

Comment:
So to validate the cert all we're doing is checking that the Serial Number matches what is on https://www.digicert.com/kb/digicert-root-certificates.htm#roots under the `DigiCert Trusted Root G4` serial. I think that is enough, but would be open to more if anyone thinks otherwise.